### ISSUE TITLE
[Select]: Placeholder WCAG contrast

### ISSUE THREAD

**Issue by nibbydev** [TEHIK-EE/tedi-design-system#103]
Insufficient contrast between select background and placeholder text

![image](https://github.com/user-attachments/assets/1b5b6e42-78b7-465b-9f51-8aa0cf5b611c)

Current contrast 3.97: ![image](https://github.com/user-attachments/assets/86b364e2-8263-48a0-9916-0a7a2d24e997)

Expected contrast 4.5: ![image](https://github.com/user-attachments/assets/c8570b5d-c09b-4aa9-8e21-8d72bf2c5769)
